books/free-programming-books-subjects.md

```diff
@@ -283,6 +284,11 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
 
 
+### Differential Privacy
+
+* [Programming Differential Privacy](https://programming-dp.com) - Joseph Near, Chiké Abuah (HTML, PDF)
```

Comment:
Super addition!
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
I suggest the for now, the best category would be Security - take a look at the other books in the category. We could also rename the category "Security and Privacy" since you can't do one without the other.